IlmBase/Config.cmake.in

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/@TARGETS_EXPORT_NAME@.cmake")
```

Comment:
From [documentation](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#generating-a-package-configuration-file):
```
check_required_components(<PackageName>) should be called at the end of the
FooConfig.cmake file ... This macro should be called even if the package
doesn’t provide any components to make sure users are not specifying components
erroneously.
```

Example:
- https://github.com/forexample/package-example/blob/00335ff3f39c3b82a4b4b6095412b5a9e3aeda81/Foo/cmake/Config.cmake.in